Create 2019-06-23-About-Georgia.md

GeorgiaHCA opened the pull request:
Created 'introducing' blog post. 
Couple of qs/editing around images still to fix :)

GeorgiaHCA committed:
Update authors.yml

Adding myself!

GeorgiaHCA committed:
Uploading images for introducing BlogPost

GeorgiaHCA committed:
Create 2019-06-23-About-Georgia.md

Created 'introducing' blog post. 
Couple of qs/editing around images still to fix :)

GeorgiaHCA committed:
Merge pull request #58 from WhitakerLab/GeorgiaHCA-patch-1

Uploading images for introducing BlogPost

GeorgiaHCA committed:
Merge pull request #54 from GeorgiaHCA/patch-1

Update authors.yml

KirstieJane committed:
KW first review, fixed a couple of typos

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,77 @@
+
+## What I do at the Turing
+
+I joined the Turing Institute for a 2-year contract, based in Kirstie’s lab, working on participatory science, sensory processing, and autism. My research interests include data ethics and the philosophy of technology. I’m currently building a citizen science platform and developing a participatory science framework.
```
I'm weirdly picky about this. Please can you put every sentence on a new line?

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,77 @@
+
+Before joining the Turing I was a civil servant, and I studied English literature for my undergrad and M.Phil degrees.
+
+I am open to collaborating and am interested in helping out with projects and papers – so get in touch if you’d like to join forces!
```
🚀 🌟

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,77 @@
+## Where I'm From
+
+<figure>
+  <img src="/images/Lauder.jpg"
```
I think this image is missing.

Also, please can you make a sub folder for the images and put them in there? Otherwise the `images/` folder is going to get really cluttered....

You don't have to move your profile picture btw (in fact, you shouldn't! It will all break if you do!)

KirstieJane replied:
Now that you've moved the images to a new folder, you need to update the paths so that this line knows where to get the image from 😸 

```suggestion
  <img src="/images/About_Georgia/Lauder.jpg"
```

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,77 @@
+
+Here I am as a tiny kid (the horse’s name is Godiva <3) – I was too young to form any memories of this moment, so I don’t know if I was happy or terrified :p
+<figure>
+  <img src="/images/Godiva.jpg"
```
This picture renders really small in the blog post - do you have a larger one?

KirstieJane replied:
Update path for image

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,77 @@
+I have a twin sister who's a professional pianist in Edinburgh. Like all twins, we share not only genes but dreams, and we have telepathic powers. Here she is playing:
+
+<figure>
+  <img src="/Images/AilsaPiano.jpg"
```
Ah - I missed this in my edits - please correct `Images/` to `images/`  (and put the picture in a sub folder)

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,77 @@
+
+I’ve spent most of my adult life in London, but I’ve also lived in Oxford (where I did my undergrad) and Cambridge (for my M.Phil), as well as in Paris and Beijing. I can tell I’ve lost my Scottish stoicism because I like my porridge with hot cream and strawberries instead of cold water. Still undecided on whether I prefer a dry martini, dry socks, and dry humour to a hot toddy, damp boots, and B ...
+
+I have a twin sister who's a professional pianist in Edinburgh. Like all twins, we share not only genes but dreams, and we have telepathic powers. Here she is playing:
```
SO COOL!

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,77 @@
+
+<figure>
+  <img src="/Images/ReadWhileYouRun.jpg">
+  <figcaption> This looks dangerous - full disclosure I've never risked this level of efficiency! Image comes from http://www.saltyrunning.com/readers-roundtable-read-run/ </figcaption>.
```
This url doesn't render in the blog post. Can you try making it an html formatted one?

```suggestion
  <figcaption> This looks dangerous - full disclosure I've never risked this level of efficiency! Image source <a href="http://www.saltyrunning.com/readers-roundtable-read-run/">here</a>.</figcaption>.
```

KirstieJane reviewed (commented):
This is fantastic! Thank you @GeorgiaHCA!

Please take a look at the reviews, they're really easy to fix 😄

Commit:
Moved pictures to subfolder

Commit:
Create Lauder.jpg

GeorgiaHCA committed:
Update 2019-06-23-About-Georgia.md

Put all sentences on a single line, slight reformatting, fig caption put into html.

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,107 @@
+
+<figure>
+  <img src="/images/ReadWhileYouRun.jpg">
+  </figcaption>This looks dangerous - full disclosure I've never risked this level of efficiency!
```
Stray `/` at the start of your figure caption meant it wasn't valid html
```suggestion
  <figcaption>This looks dangerous - full disclosure I've never risked this level of efficiency!
```

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,107 @@
+<figure>
+  <img src="/images/ReadWhileYouRun.jpg">
+  </figcaption>This looks dangerous - full disclosure I've never risked this level of efficiency!
+  Image comes from http://www.saltyrunning.com/readers-roundtable-read-run/.</figcaption>
```
Within the figure caption you have to use html to get the links to render (bit of a quirk of jekyll here).
```suggestion
 Image comes from <a href="http://www.saltyrunning.com/readers-roundtable-read-run">www.saltyrunning.com/readers-roundtable-read-run</a>
```

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,107 @@
+
+
+<figure>
+  <img src="/images/GraceHopperBugs.jpg">
```
Update path

KirstieJane reviewed (commented):
@GeorgiaHCA - I'm so sorry for abandoning this for so long!!

I've added in a couple of comments but its super close to being merged. Thanks for writing it!

GeorgiaHCA committed:
Update 2019-06-23-About-Georgia.md

Changed paths to after folder updated and amended fig caption and image source.

GeorgiaHCA committed:
Update 2019-06-23-About-Georgia.md

Typo meant Ailsa pic wasn't rendering

GeorgiaHCA commented:
Made the suggested edits :) Images show on the blog now

KirstieJane commented on _posts/blog/2019-06-23-About-Georgia.md:
```diff
@@ -0,0 +1,106 @@
+  <img src="/images/About_Georgia/ReadWhileYouRun.jpg">
+  <figcaption>This looks dangerous - full disclosure I've never risked this level of efficiency!
+  Image comes from <a href="http://www.saltyrunning.com/readers-roundtable-read-run">www.saltyrunning.com/readers-roundtable-read-run</a>
+
```
You need to close the figure tag and close the figure caption tag.

```
<figure>
  <img src="/images/About_Georgia/ReadWhileYouRun.jpg">
  <figcaption>This looks dangerous - full disclosure I've never risked this level of efficiency!
  Image comes from <a href="http://www.saltyrunning.com/readers-roundtable-read-run">www.saltyrunning.com/readers-roundtable-read-run</a>
  </figcaption>
</figure>
```

KirstieJane reviewed (commented):
Just one little change!!

KirstieJane committed:
Merge branch 'master' of https://github.com/WhitakerLab/whitakerlab.github.io into GeorgiaHCA-patch-2

GeorgiaHCA committed:
Update 2019-06-23-About-Georgia.md

figcaption sorted

GeorgiaHCA committed:
Update 2019-06-23-About-Georgia.md

extra space to make it tidier

GeorgiaHCA committed:
Update 2019-06-23-About-Georgia.md

Removing extra space
